integration-test/background/storage.js

```diff
@@ -143,5 +143,37 @@ describe('Storage blocking Tests', () => {
             await page.close()
             await teardown()
         })
+
+        it('does not block safe third party iframe JS cookies when protections are disabled', async () => {
+            // https://app.asana.com/0/1201614831475344/1203336793368587
+            const { page, bgPage, teardown } = await setup()
+            // add testPageDomain to the allowlist
+            await bgPage.evaluate(async (domain) => {
+                return new Promise((resolve, reject) => {
+                    chrome.storage.local.get('settings', ({ settings }) => {
```

Comment:
This is good, but the test will fail open if we're changing the storage layout which isn't ideal.

Could you file a follow up nit / fix to call the allowlist methods instead rather than manipulating the storage directly?